ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -362,7 +362,7 @@ tests:
     cluster_profile: aws-terraform-qe
     env:
       CASE_LABEL_FILTER: day1-supplemental&&!Exclude
-      CHANNEL_GROUP: stable
+      CHANNEL_GROUP: candidate
```

Comment:
I would suggest use stable channel group to avoid affecting from OCP version. Its testing scope should be rhcs.